kpc/admin.py

```diff
@@ -1,7 +1,45 @@
+from django import forms
 from django.contrib import admin
+from django.contrib.admin.widgets import FilteredSelectMultiple
+
+from accounts.models import Profile
+
 from .models import Licensee
 
 
+class LicenseeAdminForm(forms.ModelForm):
```

Comment:
This form has a sizable chunk of custom logic (even though it's boilerplate). Just to push back a little bit, is there an alternative that'd require less maintenance? Two ideas come to mind:

* Have we tried generating the form by spelling out the backwards relation (rather than just `__all__`, including the fields plus `contacts`)? I don't have a lot of confidence it'd work, but it'd be worth trying.
* Would it make more sense to move the field from `Profile` to `Licensee`? Then we could generate the form automatically. Of course, if we ever want to edit the connection from a `Profile` form, we're back where we started.

Another downside to adding custom logic here is that we're also on the hook for writing tests.

Reply:
Thanks! We should enumerate the fields here and include tests. 

We definitely need the ability to modify both ends of the M2M relationship from the admin